packages/permissions/src/components/authorized/authorized.tsx

```diff
@@ -122,18 +115,21 @@ const Authorized = (props: Props) => {
       { extra: namesOfNonConfiguredPermissions }
     );
 
-  const hasDemandedPermissions = props.shouldMatchSomePermissions
+  hasDemandedPermissions = props.shouldMatchSomePermissions
     ? hasSomePermissions(props.demandedPermissions, props.actualPermissions)
     : hasEveryPermissions(props.demandedPermissions, props.actualPermissions);
 
-  const hasDemandedActionRights = hasEveryActionRight(
+  hasDemandedActionRights = hasEveryActionRight(
     props.demandedActionRights || [],
     props.actualActionRights
   );
 
   return (
     <React.Fragment>
-      {props.render(hasDemandedPermissions && hasDemandedActionRights)}
+      {props.render(
+        hasDemandedDataFences ||
```

Comment:
this is where we fallback to general permissions if dataFence permissions are not met.